### PR TITLE
W2D5 curriculum specialist updates

### PR DESCRIPTION
**Pull request by Deepak-Mewada**
Checked with python 3.9 locally,
saved images to static folder

### CODE REVIEW

File: tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial1.ipynb [hunk 1]
```diff
@@ -1250,9 +768,263 @@
...
+     "model_module_version": "1.0.0",
+     "model_name": "OutputModel",
+     "state": {
+      "_dom_classes": [],
+      "_model_module": "@jupyter-widgets/output",
+      "_model_module_version": "1.0.0",
+      "_model_name": "OutputModel",
+      "_view_count": null,
+      "_view_module": "@jupyter-widgets/output",
+      "_view_module_version": "1.0.0",
```
Comment (iamzoltan) [inline tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial1.ipynb:786]: why is there widget state?

File: tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial2.ipynb [hunk 2]
```diff
@@ -1711,9 +956,618 @@
...
+   "version": "3.9.0"
+  },
+  "widgets": {
+   "application/vnd.jupyter.widget-state+json": {
+    "03575dddf5e745259cef598565ba421d": {
+     "model_module": "@jupyter-widgets/controls",
+     "model_module_version": "1.5.0",
+     "model_name": "IntSliderModel",
+     "state": {
+      "_dom_classes": [],
```
Comment (iamzoltan) [inline tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial2.ipynb:968]: same question as above

File: tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial2.ipynb [hunk 3]
```diff
@@ -1711,9 +956,618 @@
...
...kdyh2LiIhAURQMBoPaoVik+Ph4srKyLG7mK0CnTp3QaDTysOl7oCgKHTt2tMhBecn9e3Po0CGys7MtMvdDQ0OxsrJCq9WqHYrF0mq1dOrUCRsbs5p7fVsiIiLk2t8DS61yBwgLC5Pcv0darZbQ0FCLzX35znf34uLiyM3NtdjcByrF9TergSeA8ePH8/nnn/Ptt99y9OhRRo0aRU5ODk899ZTaoVUpWq2Wfv0eYsqUBSQnJ9Flz6s8kPg5eqx4r8XnZHp4UlRU8tytNDp0sGfJkuky6CSEEEIIs2YwGNBqtRZ5EwLGm9BLly5x5swZtUOxSIqiYGtrS3BwsNqh3DEXFxfat29fKW5C1aDX6y0+95OSkjh//rzaoVgkRVGwt7cnKChI7VDumJubG35+fpL7d0mv1xMZGWnRuZ+YmEhiYqLaoVgkS61yB3B3d6dt27aS+3eppMrdknP/3LlzXLx4Ue1QLJKiKDg6OhIQEKB2KHfM09OTNm3aVIrcN7uBpyFDhjBnzhwmT56Mv78/Bw4c4Pfff8fb21vt0KqMJUuW0K/fCLZuTSQ1tQYPFWfwftEPALztOA6lel/at/ejbdumAHTu3Jxff11FaGiommELIYQQQvyn48ePc+XKFYudLFMy870y3IioQVEUOnTogKOjo9qh3JXw8HC59nfp6NGjpKenW2zuV6bZr2pQFIWgoKBbLtVvCST3715CQgIZGRmS+1WUoigEBwdL7ldB8fHxZGZmSu5XUYqiWOwKF1B5ct/sBp4AXnzxRc6fP09BQQExMTGEhISoHVKV8cknn/Dyy9PJzHwAne4TgovG8rU+HYCFtGZecWfOnDlJdvYx0tI+BWDChDEWWbYqhBBCiKpj5syZzJo1C61Wi0ajsdhlnD08PGjTpg1arZY//viDoUOHqh2S2UtNTSU8PJzU1FSLrniBG7NfL1y4wKBBg9i1a5faIZm96dOnM2fOHLRaLdbW1nTs2FHtkO5K9erVadWqFVqtlt9//53hw4erHZLZu3z5MuHh4aSlpVWK3D99+jTJyckMGDBAlt66De+//z7z5s1Dq9ViY2Njsblfo0YNWrRogVarZdOmTTz22GNqh2T2Ll26REREBFevXq0UuX/y5EkuX75M//79iYqKUjskszdt2jQWLFiAVqvF1tbWYvuUvb29adasGVqtlo0bNzJixAi1Qz
```
Comment (iamzoltan) [inline tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial2.ipynb:1325]: why is this image not added like the others?

File: tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial4.ipynb [hunk 4]
```diff
@@ -268,7 +657,261 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.18"
+   "version": "3.9.0"
+  },
+  "widgets": {
+   "application/vnd.jupyter.widget-state+json": {
+    "29b02150b9a7449c8ece8b5582255586": {
+     "model_module": "@jupyter-widgets/controls",
```
Comment (iamzoltan) [inline tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial4.ipynb:665]: here too

File: tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial5.ipynb [hunk 5]
```diff
@@ -0,0 +1,1041 @@
...
+     "model_module_version": "1.0.0",
+     "model_name": "OutputModel",
+     "state": {
+      "_dom_classes": [],
+      "_model_module": "@jupyter-widgets/output",
+      "_model_module_version": "1.0.0",
+      "_model_name": "OutputModel",
+      "_view_count": null,
+      "_view_module": "@jupyter-widgets/output",
+      "_view_module_version": "1.0.0",
```
Comment (iamzoltan) [inline tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial5.ipynb:797]: same question

File: tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial6.ipynb [hunk 6]
```diff
@@ -0,0 +1,666 @@
...
...LHYbtHX6/wAfXc+J7LdRxN1z0Wa/xf8Al3+sr32bHtok32cms4a79Qf3w3q19lqnI+N5vZ32b91jfVh8Dp/i7F56pNoJGea2PTrDju/x9/rUgpeUypYLCOmIPAtl+cs8rab2NNO7gtz1vQsCkMUYLfVdeRKPl2r2AARURt+KOY/CoC2MX0jsSH+4w7xiqP0qlD8LMud9S7/I9AbW0GbMODWX8Tf4jN6lUm0eG2cCR0WuFmjS9sxF+oagX3/FRKr+kDiDzrBQakE2jn1yiwBvU7tSVoqnlZqnG5hpPYl468Z+tFa1EyE7ylJcyeQREXva7nhotplb5ugO65JHXYO4lc4orzn8MbDoNwv+uo77BVm/lHqSbmOm1Id5snAgj/e9gSLlHqQHWjp7vJJJbLcE33fbWG9TdvMze3gSXaY3eepx9QzDT1BvuXdOzLl3j7XUfwAj3f0UDl2vlcblkO++5/zOwLsn20mda8cAsSdBJvP/ABe71J+Hme/iIEix3/aJh1m4HUH2f8HrW0QuztBBt/iyH4g+BWkqdpZHSF5ZFmcGAgB9vs2NjH3+IaL9t11R488X6EevY/rv+Pr+KmqEkRdeJv5Tobb25hr13uPgSsCiNyR3Ed17fAhax+NPN+izU33O7fzdpXVDibmm4azxzf8Akp+xeCPtonbikVrdpcPVlsfELCY+1v34Fd1TXlwsWt8M3/ksfnPyj+b+qvinjUzTab0Oc4G8bjvHUVn7MYoYJ2v+6DZ462OtmHhvHa0LXNl/K3X+L/yXEP8Ayj+b/wAlIiXy1wIuDcEAgjiDqCvqqrD9uJ442sayEtjaGgubITYaC9pBuGngu/yg1Ho6f2ZPmr0EQREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREB//9k=\n",
```
Comment (iamzoltan) [inline tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial6.ipynb:349]: Whats going on with this image?

File: tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial1.ipynb (hunk 1 above)
Comment (Deepak-Mewada) [reply tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial1.ipynb:786]: Sorry, no idea.
I took that video rendering code from other tutorials.

Now I replaced the previous video rendering cell with the one given under the template  [video1 here](https://colab.research.google.com/github/neuromatch/course-content-template/blob/main/tutorials/W1D2_Template/student/W1D2_Tutorial1.ipynb#scrollTo=t9fk3pRD8dGM)

I hope it works fine now.

File: tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial3.ipynb [hunk 7]
```diff
@@ -1974,9 +649,263 @@
...
    "pygments_lexer": "ipython3",
-   "version": "3.9.18"
+   "version": "3.9.0"
+  },
+  "widgets": {
+   "application/vnd.jupyter.widget-state+json": {
+    "046054f25dad4c869bd2947642ccd869": {
+     "model_module": "@jupyter-widgets/output",
+     "model_module_version": "1.0.0",
+     "model_name": "OutputModel",
```
[the remark this answers is not in the document]
Comment (Deepak-Mewada) [reply tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial3.ipynb:659]: as above replaced the code with as given in template

File: tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial4.ipynb (hunk 4 above)
Comment (Deepak-Mewada) [reply tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial4.ipynb:665]: same , replaced

File: tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial5.ipynb (hunk 5 above)
Comment (Deepak-Mewada) [reply tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial5.ipynb:797]: same , replaced with the code as given in template

File: tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial6.ipynb [hunk 8]
```diff
@@ -0,0 +1,666 @@
...
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.0"
+  },
+  "widgets": {
+   "application/vnd.jupyter.widget-state+json": {
+    "09800842c9194a909f3623ae03e7e88c": {
+     "model_module": "@jupyter-widgets/output",
+     "model_module_version": "1.0.0",
```
[the remark this answers is not in the document]
Comment (Deepak-Mewada) [reply tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial6.ipynb:326]: sorry, not sure.

I just took the code from last year .

[the remark this answers is not in the document]
Comment (Deepak-Mewada) [reply tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial6.ipynb:326]: this tutorial is mostly same as last year's last tutorial

File: tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial6.ipynb (hunk 6 above)
Comment (Deepak-Mewada) [reply tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial6.ipynb:349]: sorry, couldn't locate the video in the tutorial. Seems like that is coming out of the video rendering cell

**Review by iamzoltan — APPROVED** [review]
LGTM!

**Review by jiapulidoar — APPROVED** [review]
Looks good! @iamzoltan Could you please merge this PR? I don't have write rights to merge, so I can update the links for W2D5. thanks!

**Pull request merged**